reframe/core/pipeline.py

```diff
@@ -348,6 +348,19 @@ def pipeline_hooks(cls):
     container_platform = variable(type(None),
                                   field=ContainerPlatformField, value=None)
 
+    #: .. versionadded:: 3.5
```

Comment:
```suggestion
    #: .. versionadded:: 3.5.0
```
I think the version should be `3.5.0`, but I am not sure.